tests/php/Forms/NumericFieldTest.php

```diff
@@ -624,70 +624,70 @@ public static function provideDataType(): array
             'int-scale-0' => [
                 'value' => 3,
                 'scale' => 0,
-                'expValue' => '3',
+                'expFormattedValue' => '3',
                 'expValueForValidation' => '3',
                 'expDataValue' => '3',
             ],
             'int-scale-1' => [
                 'value' => 3,
                 'scale' => 1,
-                'expValue' => '3',
+                'expFormattedValue' => '3.0',
```

Comment:
These tests were were previously testing getValue() and dataValue(), which are functionally identical. I changed the getValue() test to getFormattedValue()